src/common/errors.js

```diff
@@ -136,6 +136,14 @@ class EnigmaContractDataError extends Error {
   }
 }
 
+class CommitReceiptEthereumError extends Error {
```

Comment:
Where is this being used?

Reply:
Not used anymore. Thanks, it will be fixed. :)